.github/workflows/e2e.yaml

```diff
@@ -28,21 +25,30 @@ jobs:
         - sshfs.yaml
 
     steps:
-    - name: Fetch dependencies
-      run: |
-        cat >/etc/apk/repositories <<_EOF_
-        https://dl-cdn.alpinelinux.org/alpine/edge/main
-        https://dl-cdn.alpinelinux.org/alpine/edge/community
-        https://dl-cdn.alpinelinux.org/alpine/edge/testing
-        _EOF_
-        apk upgrade -Ua
-        apk add go build-base bubblewrap
     - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
-    - run: |
+    - uses: actions/setup-go@v4
+      with:
+        go-version-file: 'go.mod'
+    - name: Build package
+      run: |
+        sudo apt-get update -y
+        sudo apt-get install -y bubblewrap
+
         make melange
-        ./melange keygen && mv melange.rsa.pub /etc/apk/keys
+        ./melange keygen
         ./melange build --pipeline-dir=pipelines examples/${{matrix.example}} --arch=x86_64 --empty-workspace
 
+    - name: Check SBOM Conformance
+      run: |
+        set -euxo pipefail
+        for f in packages/x86_64/*.apk; do
+          tar -Oxf "$f" var/lib/db/sbom > sbom.json
```

Comment:
Can this have multiple files in it, or are we guaranteed that each APK only contributes its own SBOM?

Reply:
I don't think it can have multiple files, but I don't know for sure. If multiples are found they'll be concatenated in `sbom.json` and presumably something will fail.

If you have shell-fu to check that better than I am, please suggest it.